play/src/components/ConfidenceMeterCard.tsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import { useTheme } from '@mui/material/styles';
+import {
+  Typography,
+  Box,
+  RadioGroup,
+  FormControlLabel,
+  Radio,
+  FormControl,
+} from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { ConfidenceLevel, isNullOrUndefined } from '@righton/networking';
+import BodyCardStyled from '../lib/styledcomponents/BodyCardStyled';
+
+interface ConfidenceMeterCardProps {
+  selectedOption: number | null;
+  handleSelectOption: (index: number, confidence: ConfidenceLevel) => void;
+  isSelected: boolean;
+  isSmallDevice: boolean;
+  timeOfLastSelect: number | null;
+  setTimeOfLastSelect: (time: number | null) => void;
+}
+
+export default function ConfidenceMeterCard({
+  selectedOption,
+  handleSelectOption,
+  isSelected,
+  isSmallDevice,
+  timeOfLastSelect,
+  setTimeOfLastSelect,
+}: ConfidenceMeterCardProps) {
+  const theme = useTheme();
+  const { t } = useTranslation();
+  const confidenceOptionArray = [
```

Comment:
nit: Might worth to create a type or interface and then use that to annotate the type of `confidenceOptionArray `, something like:

```typescript
interface IConfidenceOption {
    text: string
    value: ConfidenceLevel
}

let confidenceOptionArray: IConfidenceOption[] = [
    {
        text: "text",
        value: ConfidenceLevel.NOT_AT_ALL
    },
]
```

Why going over this amount of hassle? It's because then you've assigned a strong type to `confidenceOptionArray` variable. This way, if anybody tries to add an item that doesn't match the others or have a typo, typescript compiler would catch that error